src/yang/ietf-twamp.yang

```diff
@@ -399,13 +399,13 @@ module ietf-twamp {
       type uint8 {
         range 10..31;
       }
-      default 15;
+      default 20;
       description
         "This parameter limits the maximum Count value, which MUST
          be a power of 2 and at least 1024 as per RFC 5357. It is
          configured by providing said power. For example,
          configuring 10 here means max count 2^10 = 1024.
-         The default is 15, meaning 2^15 = 32768.
+         The default is 20, meaning 2^15 = 1048576.
```

Comment:
This should be "The default is 20, meaning 2^20 = 1048576".

Reply:
Will fix.